### PR TITLE
live suggestions implementation

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

File: apps/suggestions/service.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    def _transform(self, item):
+        """
+        Transforms an item in dictionary form to plain text.
+        :param item: dict
+        :return: str
+        """
+        text = ''
+        fields = ['slugline', 'headline', 'body_html', 'body_text', 'description_text']
+        for field in fields:
```
Comment (petrjasek) [inline apps/suggestions/service.py]: imo a bit more pythonic would be: `return '\n\n'.join([item[field] for field in fields])`

Comment (ioanpocol) [inline apps/suggestions/service.py]: return '\n\n'.join([item[field] for field in fields if field in item])

File: apps/suggestions/service.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        keywords = self.provider.get_keywords(self._transform(item))
+        if not keywords:
+            return ElasticCursor([])
+
+        query = {
+            'query': {
+                'filtered': {
+                    'query': {
+                        'query_string': {
+                            'query': ' '.join(kwd['text'] for kwd in keywords)
```
Comment (ioanpocol) [inline apps/suggestions/service.py]: The keywords comes with a relevance factor between 0 and 1. You can use this factor on elastic search in order to improve the relevance of the returned content items.

**Comment by ioanpocol** [comment]
@mugurrus  Please rebase this PR

File: apps/suggestions/service.py (hunk 2 above)
Comment (mugurrus) [inline apps/suggestions/service.py]: In another PR

**Pull request merged**